.github/workflows/issues.yml

```diff
@@ -0,0 +1,20 @@
+name: Add issue or pull request to Project
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request_target:
+    types:
+      - opened
+      - reopened
```

Comment:
I like being able to look at the pull requests in the project, you can filter by `is:pr` or `is:issue`, since all pull requests might not have an issue associated with them. I think it helps ensure that pull requests don't get accidentally ignored or missed.